src/get-config.c

```diff
@@ -0,0 +1,44 @@
+/* 
+ * Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+ * Copyright (c) 2015-2017, Gregory M. Kurtzer. All rights reserved.
+ * Copyright (c) 2017, Vanessa Sochat All rights reserved.
```

Comment:
Just to REALLY nitpick, Add a period after Vanessa Sochat so that it's unified in formatting with the other lines :) 
`* Copyright (c) 2017, Vanessa Sochat. All rights reserved.`